server/game/cards/12-2T2D/Bedazzle.js

```diff
@@ -0,0 +1,38 @@
+const SpellCard = require('../../spellcard.js');
+
+class Bedazzle extends SpellCard {
+    setupCardAbilities(ability) {
+        this.spellAction({
+            title: 'Bedazzle',
+            playType: 'cheatin resolution',
+            cost: ability.costs.bootSelf(),
+            difficulty: 3,
+            choosePlayer: true,
+            onSuccess: (context) => {
+                context.chosenPlayer.modifyRank(-2, context);
+                this.game.addMessage('{0} uses {1} to lower {2}\'s draw hand by 2 ranks; Current rank is {3}',
+                    context.player, this, context.chosenPlayer, context.chosenPlayer.getTotalRank());
+                if(this.game.shootout) {
+                    this.game.promptForSelect(context.player, {

```

Comment:
In case you are selecting another target in the handler, you should use `context.ability.selectAnotherTarget`.
```suggestion
                    context.ability.selectAnotherTarget(context.player, context, {

```